src/controls/IRoomControllerV2.ts

```diff
@@ -0,0 +1,380 @@
+            uuid + '.absentMaxOffset',
+            {
+                name: control.name + ': absentMaxOffset',
+                read: true,
+                write: absentMaxOffsetWrite,
+                type: 'number',
+                role: absentMaxOffsetWrite ? 'level.temperature' : 'value.temperature',
+            },
+            control.states.absentMaxOffset,
+            async (name: string, value: any) => {
+                await this.setStateAck(name, value);
+                await this.updateTempTargetMinMax();
+            },
+        );
+        this.addStateChangeListener(
+            uuid + '.absentMaxOffset',
+            (oldValue: OldStateValue, newValue: CurrentStateValue) => {
+                this.sendCommand(control.uuidAction, 'setAbsentMaxTemperature/' + newValue);
+            },
+        );
+
+        await this.updateStateObjectAsync(
+            uuid + '.comfortTemperatureOffset',
+            {
+                name: control.name + ': comfortTemperatureOffset',
+                read: true,
+                write: true,
+                type: 'number',
+                role: 'level.temperature',
+            },
+            control.states.comfortTemperatureOffset,
+            async (name: string, value: any) => {
+                await this.setStateAck(name, value);
+                await this.updateTempTargetMinMax();
+            },
+        );
+        this.addStateChangeListener(
+            uuid + '.comfortTemperatureOffset',
+            (oldValue: OldStateValue, newValue: CurrentStateValue) => {
+                this.sendCommand(control.uuidAction, 'setComfortModeTemp/' + newValue);
+            },
+        );
+
+        await this.updateStateObjectAsync(
+            uuid + '.tempTarget',
+            {
+                name: control.name + ': tempTarget',
+                read: true,
+                write: true,
+                type: 'number',
+                role: 'level.temperature',
+            },
+            control.states.tempTarget,
+            async (name: string, value: any) => {
+                await this.setStateAck(name, value);
+                await this.updateTempTargetMinMax();
+            },
+        );
+        this.addStateChangeListener(uuid + '.tempTarget', (oldValue: OldStateValue, newValue: CurrentStateValue) => {
+            // Target has been set manually so start an override at that value
+            // setManualTemperature is redundant in this context as simply setting that value
+            // does not actually tell IRC to switch to manual mode.
+            // We could force whatever updates the tempTarget to then need to change mode to manual
+            // but that would be illogical as the target now shown would not actually be the target
+            // being worked with.
+            // So because of this, just start an override at the given temperature.
+            this.sendCommand(control.uuidAction, 'override/3//' + newValue);
```

Comment:
This is a strange implementation by Loxone, but I believe we should stick to their way of working:
- tempTarget should only set the target temperature
- a writable flag "override" should then enable/disable the manual overriding (enable of course still without timer)
- stopOverride "button" below should be removed

Reply:
No.

I am basically trying to add all the controls that are available in the Loxone app. It has a 'stopOverride' button so why would we not present it here?